### PR TITLE
FP hash on the GUI

### PR DESCRIPTION
**Pull request by VivaanBahl**

### CODE REVIEW

**Comment by VasuAgrawal** [comment]
Is this low level fingerprint hash?

**Comment by VivaanBahl** [comment]
yes

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java [hunk 1]
```diff
@@ -296,11 +298,12 @@ public synchronized void run() {
 
                 //block until we have a message from the queue
                 Message toSort;
+                String topic;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java]: Codacy Issue found: [Avoid unused local variables such as 'topic'.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2346639260/issues/source?bid=2938665&fileBranchId=3216355#l301)

**Comment by TrevorDecker** [comment]
I am singing off on this

**Comment by VasuAgrawal** [comment]
Don't pull this. It uses the same topic logging system that was closed in an earlier PR; wait for my pull request to go in and update this to work with that.

**Comment by VivaanBahl** [comment]
Yeah I'm going to cherry pick that out sometime later. I accidentally based this branch off the wrong one

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java [hunk 2]
```diff
@@ -298,9 +300,8 @@ public synchronized void run() {
                 Message toSort;
                 try {
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java]: Codacy Issue found: [Avoid unused local variables such as 'topic'.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2348014431/issues/source?bid=2938665&fileBranchId=3216355#l301)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java [hunk 3]
```diff
@@ -328,10 +329,10 @@ public synchronized void run() {
                         steeringHits++;
                     } else if(toSort instanceof ImageMessage){
                         imageHits++;
-                    } else {
+                    }// else {
                         //a new kind of message!
-                        new RobobuggyLogicNotification("New message came in that we aren't tracking", RobobuggyMessageLevel.WARNING);
-                    }
+//                        new RobobuggyLogicNotification("New message came in that we aren't tracking", RobobuggyMessageLevel.WARNING);
+//                    }
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java]: why did you take this out?

**Comment by TrevorDecker** [comment]
I am ok with this once @ramensandwich  checks it off

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java (hunk 2 above)
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java]: Codacy Issue found: [Avoid unused local variables such as 'topic'.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2370142059/issues/source?bid=2938665&fileBranchId=3216355#l301)

**Comment by ramensandwich** [comment]
I have tested this and it seems like it works.  Some text is overlapping on the gui, however, and I'd like that to be cleaned up.  Once the merge conflicts and overlapping text have been resolved, I will merge this into master.

**Pull request merged**